public/app/components/codeEditor/codeEditor.js

```diff
@@ -1,22 +1,29 @@
 angular.module('kibibitCodeEditor')
 
-.directive('codeEditor', function() {
+.directive('kbCodeEditor', function() {
   return {
     scope: {},
     bindToController: {
-      code: '='
```

Comment:
if you changed the code editor to get a file path, you can completely remove the `code` attribute (from here that is)